html/index.html

```diff
@@ -0,0 +1,201 @@
+        <div class="header-logo"></div>
+        <h1 class="main-title">Seattle's Premiere dog adoption agency</h1>
+        <div class="checkout-balance">
+            <img class="cart-icon" src="../images/cart-icon.svg" alt="header image">
+            <span class="dollar">$1234.56</span>
+        </div>
+        <a class="button-checkout" href="checkout.html">checkout</a>
+    </header>
+
+    <nav class="topnav">
+        <a class="navigation-link" href="index.html">Home</a> /
+        <a class="navigation-link" href="all-dogs.html">Dogs</a> /
+        <a class="navigation-link" href="blog.html">Blog</a>
+    </nav>
+
+    
+        
+        <main class="home-main">
+
+            <aside class="blog-preview">
+                <h3 class="title">From our blog</h3>
+
+                <div class="blog-preview-snippet">
+                    <img class="picture" src="../small-images/woman-with-dog-canyon-smaller.jpg" alt="woman with dog canyon">
+                    <h4 class="subtitle">
+                        Traveling with your dog
+                    </h4>
+
+                    <p class="summary">
+                        Iciustiam, et ad quiaes cum que
+                        excea sinciendi qui offic temquas
+                        doluptios dolupta tiandebis nem ut.
+                    </p>
+
+                    <a class="read-more" href="blog.html">Read more</a>
+
+
+                </div> <!-- end of blog-preview-snippet  -->
+
+                <div class="blog-preview-snippet">
+                    <img class="picture" src="../small-images/three-dogs-smaller.jpg" alt="three dogs">
+                    <h4 class="subtitle">
+                        How to walk multiple dogs
+                    </h4>
+
+                    <p class="summary">
+                        Iciustiam, et ad quiaes cum que
+                        excea sinciendi qui offic temquas
+                        doluptios dolupta tiandebis nem ut.
+                    </p>
+
+                    <a class="read-more" href="blog.html">Read more</a>
+
+
+                </div> <!-- end of blog-preview-snippet  -->
+
+                <div class="blog-preview-snippet">
+                    <img class="picture" src="../small-images/woman-playing-with-dog-smaller.jpg" alt="woman playing with dog">
+                    <h4 class="subtitle">
+                       Teach your dog to fetch!
+                    </h4>
+
+                    <p class="summary">
+                        Iciustiam, et ad quiaes cum que
+                        excea sinciendi qui offic temquas
+                        doluptios dolupta tiandebis nem ut.
+                    </p>
+
+                    <a class="read-more" href="blog.html">Read more</a>
+
+
+                </div> <!-- end of blog-preview-snippet  -->
+
+
+            </aside>
+            
+
+            <section class="home-section">
+                <div class="introduction">
+                    <img class="intro-image" 
```

Comment:
Your intro image does not scale properly. It looks tiny on desktop, then enlarges on mobile.